### PR TITLE
Add Hacking Wth Swift

### PR DESCRIPTION
**Pull request by naeemshaikh90**
## Project URL
https://www.hackingwithswift.com

## Description
Learn to code iPhone and iPad apps with 3 Swift tutorials.
 
## Checklist

- [x] Only one project/change is in this pull request
- [x] Addition in chronological order (bottom of category)
- [x] Supports iOS 8 or later
- [x] Supports Swift 3
- [ ] Has a commit from less than 2 years ago
- [ ] Has a **clear** README in English

### CODE REVIEW

**Comment by danger-awesome-ios** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>

    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="true">Found 2 link issues</td>
    </tr>
  </tbody>
</table>

#### Link issues by [`awesome_bot`](https://github.com/dkhamsing/awesome_bot)

  Line | Status | Link
| ---: | :----: | --- |
1912 | [301](https://httpstatuses.com/301) | https://github.com/movielala/VideoSplashKit redirects to<br>https://github.com/mojilala/VideoSplashKit
2156 | [301](https://httpstatuses.com/301) | https://www.reddit.com/r/iosprogramming redirects to<br>https://www.reddit.com/r/iOSProgramming/


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

**Comment by lfarah** [comment]
Hey @naeemshaikh90, thanks for the contribution! 🎉 

Can you please just change the title from ```HACKING WITH SWIFT``` to ```Hacking with Swift``` ?

Thanks! 😃

**Comment by naeemshaikh90** [comment]
@dkhamsing invalid reason?

**Comment by dkhamsing** [comment]
Please change the title in the readme 
Should not be all caps

File: README.md [hunk 1]
```diff
@@ -1909,7 +1909,7 @@ Most of these are paid services, some have free tiers.
 * [Instructions](https://github.com/ephread/Instructions) - Easily add customizable coach marks into you iOS project. :large_orange_diamond:
 * [SwiftyWalkthrough](https://github.com/ruipfcosta/SwiftyWalkthrough) - The easiest way to create a great walkthrough experience in your apps, powered by Swift. :large_orange_diamond:
 * [Gecco](https://github.com/yukiasai/Gecco) - Spotlight view for iOS. :large_orange_diamond:
-* [VideoSplashKit](https://github.com/movielala/VideoSplashKit) - Video based UIViewController :large_orange_diamond:
```
Comment (dkhamsing) [inline README.md]: please undo this change

File: README.md [hunk 2]
```diff
@@ -2153,7 +2153,7 @@ Most of these are paid services, some have free tiers.
 * [ASCIIwwdc](http://asciiwwdc.com)
 * [Natasha The Robot](https://www.natashatherobot.com/)
 * [Apple's Swift Blog](https://developer.apple.com/swift/blog/) :large_orange_diamond:
-* [iOS Programming Subreddit](https://www.reddit.com/r/iosprogramming)
```
Comment (dkhamsing) [inline README.md]: please undo this change

File: README.md (hunk 1 above)
Comment (naeemshaikh90) [inline README.md]: Will you help me to undo this change?

**Comment by dkhamsing** [comment]
Why don't you start over with a new pull request 
Make sure you start from https://github.com/vsouza/awesome-ios
